cluster-operator/src/test/java/io/strimzi/operator/cluster/ClusterOperatorConfigTest.java

```diff
@@ -44,29 +45,29 @@ public void testDefaultConfig() {
 
         ClusterOperatorConfig config = ClusterOperatorConfig.fromMap(envVars);
 
-        assertEquals(singleton("namespace"), config.getNamespaces());
-        assertEquals(ClusterOperatorConfig.DEFAULT_FULL_RECONCILIATION_INTERVAL_MS, config.getReconciliationIntervalMs());
-        assertEquals(ClusterOperatorConfig.DEFAULT_OPERATION_TIMEOUT_MS, config.getOperationTimeoutMs());
+        assertThat(config.getNamespaces(), is(singleton("namespace")));
+        assertThat(config.getReconciliationIntervalMs(), is(ClusterOperatorConfig.DEFAULT_FULL_RECONCILIATION_INTERVAL_MS));
+        assertThat(config.getOperationTimeoutMs(), is(ClusterOperatorConfig.DEFAULT_OPERATION_TIMEOUT_MS));
     }
 
     @Test
     public void testReconciliationInterval() {
 
         ClusterOperatorConfig config = new ClusterOperatorConfig(singleton("namespace"), 60_000, 30_000, false, new KafkaVersion.Lookup(emptyMap(), emptyMap(), emptyMap(), emptyMap()), null, null);
 
-        assertEquals(singleton("namespace"), config.getNamespaces());
-        assertEquals(60_000, config.getReconciliationIntervalMs());
-        assertEquals(30_000, config.getOperationTimeoutMs());
+        assertThat(config.getNamespaces(), is(singleton("namespace")));
+        assertThat(config.getReconciliationIntervalMs(), is(60_000L));
+        assertThat(config.getOperationTimeoutMs(), is(30_000L));
```

Comment:
Wouldn't be better have some Constants which express `defaultOperatiorTimeMs`? What do you think ? Same with `defaultReconcilationIntervalMs`